draft-ietf-opsawg-rfc5706bis.md

```diff
@@ -1250,7 +1250,9 @@ DM         DM        DM     --> concrete/detailed model
 ##  Security Management {#sec-security-mgmt}
 
    Protocol Designers should consider how to monitor and manage security
-   aspects and vulnerabilities of the New Protocol or Protocol Extension.
+   aspects and vulnerabilities of the New Protocol or Protocol Extension,
+   including periodically re-assessing the design of the New Protocol or
```

Comment:
While I'm all for security, what does this even mean in practical terms? 
"periodically re-assessing the design of the New Protocol or Protocol Extension as vulnerabilities arise"... 
Is this a security considerations suggestion, or an operational considerations one? 
To me, this crosses the line and it is a security thing and not an Ops one... 
As such, I think it not only does not add, but actually removes from the Ops focus...